.travis.yml

```diff
@@ -1 +1,9 @@
 language: java
+install:
+  - mvn clean install -DskipTests
```

Comment:
@wluyima the default Travis' istallation step is `mvn install -DskipTests`, tests are being run in next step by `mvn test`